react/src/views/Box/BoxView.tsx

```diff
@@ -200,6 +201,22 @@ function BTBox() {
 
   const boxData = allData.data?.box;
 
+  useEffect(() => {
+    setCurrentState(boxData?.state);
+    const shipmentId = boxData?.shipmentDetail?.shipment.id;
+    // open reconciliation overlay if the box state is receiving
+    if (shipmentId && boxData?.state === BoxState.Receiving) {
```

Comment:
I think here should also be a check if you are currently in the target base. Otherwise, the reconciliation ui also can be added by the sending org or maybe it is reconciled to a wrong base by a user who has access to multiple bases.

I will post about this on slack and start a discussion